### ISSUE TITLE
explorer-*/bin/start.sh does not have _BPX_JOBNAME

### ISSUE THREAD

**Issue by jthyssenrocket** [zowe/zowe-install-packaging#1319]
It looks like {exlorer-jes,explorer-mvs,explorer-uss}/bin/start.sh is missing the _BPX_JOBNAME logic. 
This causes the address space name to be incorrect (which in turn can cause network issues if a certain port is reserved for a specific address space)

My suggestion is to add similar logic as the other start.sh scripts have. This works on my system: 

```
# start service
COMPONENT_CODE=UU
_BPX_JOBNAME=${ZOWE_PREFIX}${COMPONENT_CODE} $NODE_BIN $SERVER_DIR/src/index.js \

```

**Comment by stevenhorsman** [comment]
Duplicate of https://github.com/zowe/zlux/issues/393 and I think that the explorer components have been fixed in the 1.11 release based on the comments in there.

**Comment by jthyssenrocket** [comment]
Thanks! It does appear to be a duplicate
